lib/internal/modules/cjs/loader.js

```diff
@@ -794,9 +794,13 @@ Module.runMain = function() {
       return loader.import(pathToFileURL(process.argv[1]).pathname);
     })
     .catch((e) => {
+      // TODO(addaleax): This should, ideally, trigger a fatal exception
```

Comment:
you can use `triggerFatalException` from `internalBinding('util')`

Reply:
@devsnek Thanks, it’s much better now. Done!